src/System.Text.RegularExpressions/src/System.Text.RegularExpressions.csproj

```diff
@@ -47,13 +53,16 @@
     <Compile Include="$(CommonPath)\CoreLib\System\Collections\Generic\ValueListBuilder.cs">
       <Link>Common\System\Collections\Generic\ValueListBuilder.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\CoreLib\System\Collections\HashtableExtensions.cs">
+      <Link>Common\System\Collections\HashtableExtensions.cs</Link>
+    </Compile>
   </ItemGroup>
   <!-- Files that enable compiled feature -->
   <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp'">
-    <Compile Include="System\Text\RegularExpressions\CompiledRegexRunnerFactory.cs" />
-    <Compile Include="System\Text\RegularExpressions\CompiledRegexRunner.cs" />
-    <Compile Include="System\Text\RegularExpressions\RegexCompiler.cs" />
-    <Compile Include="System\Text\RegularExpressions\RegexLWCGCompiler.cs" />
+    <Compile Include="System\Text\RegularExpressions\Runner\CompiledRegexRunnerFactory.cs" />
```

Comment:
Coding conventions says that directory should match the namespace. There is no System.Text.RegularExpressions.Runner namespace...

Reply:
I didn't know that we are enforcing that. Folders especially here make sense (as we don't have an sub namespaces) but I'm fine with flattening it back.